clustered/integration-test/build.gradle

```diff
@@ -19,24 +19,23 @@ configurations {
 }
 
 dependencies {
-  // not required by gradle but required by the IDE because 'dist' does not have any transitive dependencies
-  testCompileOnly project(':clustered:client')
-  testCompileOnly project(':clustered:common')
-  testCompileOnly project(':impl')
-  testCompileOnly project(':xml')
-  testCompileOnly "org.terracotta.internal:client-runtime:$terracottaCoreVersion"
-  testCompileOnly "org.terracotta:runnel:$terracottaPlatformVersion"
-  testCompileOnly "org.terracotta:lease-api:$terracottaPlatformVersion"
+  testCompile project(':clustered:client')
```

Comment:
So do we have *any* tests that run against the final kit? How would we know if we broke it?